solr/core/src/java/org/apache/solr/cloud/OverseerTaskProcessor.java

```diff
@@ -129,17 +128,19 @@ public String toString() {
 
   private final Object waitLock = new Object();
 
-  protected OverseerMessageHandlerSelector selector;
-
   private OverseerNodePrioritizer prioritizer;
 
   private String thisNode;
 
+  private OverseerCollectionMessageHandler overseerCollectionMessageHandler;
```

Comment:
I suggest simply calling this "messageHandler" as it isn't ambiguous.

Reply:
Okay, will change that.